README.md

```diff
@@ -171,6 +171,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 | CoinBin | Cryptocurrency information | No | Yes | Unknown | [Go!](https://coinbin.org/) |
 | CoinDesk | Bitcoin Price Index | No | No | Unknown | [Go!](http://www.coindesk.com/api/) |
 | Coinigy | Interacting with Coinigy Accounts and Exchange Directly | `apiKey` | Yes | Unknown | [Go!](https://coinigy.docs.apiary.io) |
+| CoinLayer | Real-time Crypto Currency Exchange Rates JSON Rest API | `apiKey` | Yes | Unknown | [Go!](https://coinlayer.com) |
```

Comment:
Please remove "API" from the description. Everything here is an API so it adds nothing of value :)